.github/workflows/test-pr.yml

```diff
@@ -0,0 +1,33 @@
+name: Test PR
+
+on:
+  pull_request:
+    branches: [ master ]
+jobs:
+  # This workflow contains a single job called "build"
+  build:
+    # The type of runner that the job will run on
+    runs-on: ubuntu-latest
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+    # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+    - uses: actions/checkout@v2
+    - name: Set up Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: '3.x'
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip setuptools wheel
+        pip install -r requirements-dev.txt
```

Comment:
Hey @triptec, do you mind adding `pip install  "sqlalchemy<1.4"` below this line since we dealing with 1.3 first before adding support in https://github.com/absent1706/sqlalchemy-mixins/pull/51